### PR TITLE
Add spark fuzzer to nightly job.

### LINKED ISSUE #3257: Add CI jobs to run fuzzer on SparkSQL functions
CC: @kagamiori @pedroerp @bikramSingh91 

We have CI jobs to run fuzzer on PrestoSQL functions: both on each PR and nightly. We need to add similar jobs for SparkSQL functions.

### PR DESCRIPTION
**Pull request by kgpai**
This job runs for 10 minutes, and stores the resultant logs as artifacts. The artifacts can be accessed via the artifacts section in circleci for the particular instance of the run. We also rename linux-fuzzer-run to linux-presto-fuzzer-run.

### CODE REVIEW

File: scripts/setup-ubuntu.sh [hunk 1]
```diff
@@ -83,9 +83,16 @@ function install_folly {
   cmake_install -DBUILD_TESTS=OFF
 }
 
+function install_conda {
```
Comment (mbasmanova) [inline scripts/setup-ubuntu.sh:86]: This change seems not relevant. Can it be extracted into a separate PR?

File: .circleci/config.yml [hunk 2]
```diff
@@ -39,6 +39,7 @@ workflows:
     jobs:
       - doc-gen-job
       - linux-fuzzer-run
+      - linux-spark-fuzzer-run
```
Comment (mbasmanova) [inline .circleci/config.yml:42]: Should we rename linux-fuzzer-run to linux-presto-fuzzer-run?

**Review by mbasmanova — COMMENTED** [review]
@kgpai Thank you for adding this.

The PR description repeats the PR title. Let's remove this repetition. Also, consider expanding the PR description to explain how long the fuzzer runs for and what artifacts are preserved and how they can be acceessed.

File: scripts/setup-ubuntu.sh (hunk 1 above)
Comment (kgpai) [reply scripts/setup-ubuntu.sh:86]: Thanks Mahsa, this is from another PR, rebasing with main will remove this.

**Review by mbasmanova — APPROVED** [review]
